services/offline/src/lib/clear-sensitive-caches.ts

```diff
@@ -5,7 +5,6 @@ export const SECTIONS_STORE = 'sections-store'
 // Non-sensitive caches that can be kept:
 const KEEPABLE_CACHES = [
     /^workbox-precache/, // precached static assets
-    /^app-shell/, // will be immediately overwritten by network-first strategy
     /^other-assets/, // static assets cached at runtime - shouldn't be sensitive
```

Comment:
Probably fine, but is there a reason to exclude `other-assets`?

Reply:
Just for a small performance benefit - these are static assets served with an SWR strategy, so would make things like map tiles load faster, and I couldn’t think of a way that static assets could be user-specific or sensitive

Reply:
@amcgee thoughts?

Reply:
Sounds fine!